pkg/apis/controller/addtoscheme_pytorchjob_v1.go

```diff
@@ -12,11 +12,12 @@ limitations under the License.
 
 package apis
 
-import (
-	"github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1"
-)
+// import v1 "github.com/kubeflow/pytorch-operator/pkg/apis/pytorch/v1"
+// TODO (andreyvelich): We need to update common library for PyTorch operator
+// It uses old path for controller
+// https://github.com/kubeflow/pytorch-operator/blob/master/pkg/controller.v1/pytorch/status.go#L28
 
 func init() {
 	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
-	AddToSchemes = append(AddToSchemes, v1.SchemeBuilder.AddToScheme)
+	// AddToSchemes = append(AddToSchemes, v1.SchemeBuilder.AddToScheme)
```

Comment:
Should we keep the comment now?

Reply:
@gaocegege Do you want to remove this ?
Currently we can't use PyTorch operator lib